Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -2579,6 +2579,21 @@ class ByteBufferTest: XCTestCase {
         self.buf.moveWriterIndex(forwardBy: 5)
         XCTAssertEqual("hellohello", self.buf.readString(length: 10))
     }
+    
+    func testRepeatingBytes() {
+        
```

Comment:
we could make this test more readable by writing:

```
func testRepeatingBytes() {
    let written = self.buf.writeRepeatingByte(9, count: 1024)
    XCTAssertEqual(1024, written)
    XCTAssertEqual(Array(repeating: UInt8(9), count: 1024), Array(self.buf.readableBytesView))
}
```

- we should also add a test case that tests the edge condition (`count: 0`)

Reply:
Looks good, I'll also bump the number of bytes up I think